daisy_workflows/build-publish/sqlserver/sql-2022-preview-windows-2022-dc-uefi.publish.json

```diff
@@ -0,0 +1,45 @@
+{{/*
+  Template to publish SQL Server Windows images
+  By default this template is setup to publish to the 'gce-image-builder'
+  project, the 'environment' variable can be used to publish to 'test', 'prod',
+  or 'staging'.
+  DeleteAfter is set to 180 days for all environments other than prod where no
+  time period is set.
+*/}}
+{
+  "Name": "sql-server-windows-uefi",
+  "SourceGCSPath":   "gs://gce-image-archive/sqlserver-uefi",
+  {{$work_project := `"gce-image-builder"` -}}
+  {{$endpoint := `"https://www.googleapis.com/compute/alpha/projects/"` -}}
+  {{$delete_after := `"24h*30*6"` -}}
+  {{if eq .environment "test" -}}
+  "WorkProject": {{$work_project}},
+  "PublishProject": "bct-prod-images",
+  "ComputeEndpoint": {{$endpoint}},
+  "DeleteAfter": {{$delete_after}},
+  {{- else }}
+  "WorkProject": "bct-staging-images",
+  "PublishProject": "bct-staging-images",
+  "ComputeEndpoint": "https://www.googleapis.com/compute/staging_alpha/projects/",
+  "DeleteAfter": {{$delete_after}},
+  {{$guest_features := `["MULTI_IP_SUBNET","UEFI_COMPATIBLE","VIRTIO_SCSI_MULTIQUEUE","WINDOWS"]` -}}
+  {{$time := trimPrefix .publish_version "v"}}
+  "Images": [
+    {
+      "Prefix": "sql-2022-preview-windows-2022-dc",
+      "Family": "sql-ent-2022-win-2022",
+      "Description": "Microsoft, SQL Server 2022 Preview, on Windows Server 2022 Datacenter, x64 built on {{$time}}, supports Shielded VM features",
+      "Architecture": "X86_64",
+      "Licenses": [
+        {{if eq .environment "staging" -}}
+        "projects/windows-sql-cloud/global/licenses/sql-server-2022-express",
```

Comment:
Is this inappropriate? I didn't see any sql licenses at all in bct-staging-functional.

Reply:
No, you'll need to create a sql-server-2022-express in bct-staging-functional

Reply:
Turns out I'd already done this, so I've updated it.

Reply:
I don't see any update to the PR, did you forget to push a change?

Reply:
That I did. Pushed it now.